share/completions/git.fish

```diff
@@ -1186,13 +1186,13 @@ complete -c git -n '__fish_git_using_command log' -l no-min-parents -d 'Show onl
 complete -c git -n '__fish_git_using_command log' -l no-max-parents -d 'Show only commit without a maximum number of parents'
 complete -c git -n '__fish_git_using_command log' -l first-parent -d 'Follow only the first parent commit upon seeing a merge commit'
 complete -c git -n '__fish_git_using_command log' -l not -d 'Reverse meaning of ^ prefix'
-complete -c git -n '__fish_git_using_command log' -l all -d 'Pretend as if all refs in refs/ are listed on the command line as <commit>'
-complete -f -c git -n '__fish_git_using_command log' -l branches -d 'Pretend as if all refs are in refs/heads are listed on the command line as <commit>'
-complete -f -c git -n '__fish_git_using_command log' -l tags -d 'Pretend as if all refs are in ref/tags are listed on the command line as <commit>'
-complete -f -c git -n '__fish_git_using_command log' -l remotes -d 'Pretend as if all refs in refs/remotes  are listed on the command line as <commit>'
-complete -x -c git -n '__fish_git_using_command log' -l glob -d 'Pretend as if all refs matching shell glob are listed on the command line as <commit>'
+complete -c git -n '__fish_git_using_command log' -l all -d 'Pretend that all refs in refs/ are listed on the cmd line as <commit>'
+complete -f -c git -n '__fish_git_using_command log' -l branches -d 'Pretend that all refs in refs/heads are listed on the cmd line as <commit>'
```

Comment:
This would then be "Show log for all matching branches" - it can take an optional argument to match branches.